server/game/cards/attachments/02/oldbearsraven.js

```diff
@@ -0,0 +1,24 @@
+const DrawCard = require('../../../drawcard.js');
+
+class OldBearsRaven extends DrawCard {
+    setupCardAbilities(ability) {
+        this.whileAttached({
+            match: card => card.name === 'Old Bear Mormont',
+            effect: ability.effects.addIcon("intrigue")
+        });
+        this.whileAttached({
+             effect: ability.effects.addKeyword("stealth")
```

Comment:
Per the project's eslint rules, strings should be in single quotes (stealth here, intrigue above). Also missing a semi-colon after the whileAttached declaration which eslint will complain about. Otherwise, looks good. 👍